VideoWeb/VideoWeb/ClientApp/src/app/shared/footer/footer.component.html

```diff
@@ -1,56 +1,45 @@
-            </a>
-          </li>
-        </ul>
-
-        <svg
-          role="presentation"
-          focusable="false"
-          class="govuk-footer__licence-logo"
-          xmlns="http://www.w3.org/2000/svg"
-          viewBox="0 0 483.2 195.7"
-          height="17"
-          width="41"
-        >
-          <path
-            fill="currentColor"
-            d="M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145"
-          ></path>
-        </svg>
-        <span class="govuk-footer__licence-description">
-          <span i18n="@@footer_p_1">{{ 'footer.all-content-availabe-under' | translate }} </span>
-          <a
-            class="govuk-footer__link"
-            href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
-            rel="license"
-          >{{ 'footer.open-government-license' | translate }}</a>
-          <span>{{ 'footer.except' | translate }}</span>
-        </span>
-      </div>
-      <div class="govuk-footer__meta-item">
-        <a
-          class="govuk-footer__link govuk-footer__copyright-logo"
-          href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/"
-          >{{ 'footer.copyright' | translate }}</a
-        >
-      </div>
+    <div class="govuk-width-container">
+        <div class="govuk-footer__meta">
+            <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+                <ul class="govuk-footer__inline-list">
+                    <li class="govuk-footer__inline-list-item">
+                        <a class="govuk-footer__link" [routerLink]="[privacyPolicyUri]">
+                            {{ 'footer.privacy-policy' | translate }}
+                        </a>
+                    </li>
+                    <li class="govuk-footer__inline-list-item">
+                        <a class="govuk-footer__link" [routerLink]="[accessibilityUri]">
+                            {{ 'footer.accessibility' | translate }}
+                        </a>
+                    </li>
+                    <li class="govuk-footer__inline-list-item">
+                        <a class="govuk-footer__link" id="translateSwitch" [routerLink]="" (click)="switchLaguage();">
+                            {{ 'footer.translate-to' | translate }}
+                        </a>
+                    </li>
+                </ul>
+                <svg role="img" aria-label="open goverment licence logo" focusable="false" class="govuk-footer__licence-logo"
```

Comment:
This needs a Welsh translation as well.

Reply:
Also government is spelled wrong

Reply:
It should also probably be an alt rather than an aria-label as this is an image.

Reply:
Last one I swear. But we should confirm one of the following for the English before getting Welsh translation. I don't know who would make this call perhaps discuss with Neil or Lorenzo.
Open Government Licence logo
O G L logo
O G L (Open Government Licence) logo

Regarding the acronyms having spaces: https://accessibility.psu.edu/abbreviations/

Reply:
I spoke to Neil and the OGL logo needs an empty alt as it is purely decorative. I added a translation for the "Copyright Information" string.